SEED-platform/seed
# 3295 column desc field memo box

#### Any background context you want to provide?
The column description field in the column settings page was too narrow.

#### What's this PR do?
Set minimum width to 350 pixels.

#### How should this be manually tested?
Go to column settings page in an organization and check that the column description field is wide enough.

#### What are the relevant tickets?
#3295 

#### Screenshots (if appropriate)
